packs/AWS_CloudTrail.sql

```diff
@@ -0,0 +1,201 @@
+-- varmap:
+--   base_name: 'AWS_CLOUDTRAIL_{connection_name}_EVENTS'
+--   stage: 'data.{base_name}_STAGE'
+--   staging_table: 'data.{base_name}_STAGING'
+--   landing_table: 'data.{base_name}_CONNECTION'
+--   pipe = 'data.{base_name}_PIPE'
+--   task: 'data.{base_name}_TASK'
+--   stream: 'data.{base_name}_STREAM'
+--   storage_integration: 'data.{base_name}_STORAGE_INTEGRATION'
+
+
+CREATE SEQUENCE IF NOT EXISTS LANDING_TABLE_SEQ START 1 INCREMENT 1;
```

Comment:
let's keep the casing consistent -- we want our names to be lower case, so `LANDING_TABLE_SEQ` should be, as well, to separate it from the language constructs